posts/2020-09-30-microprofile-language-support-liberty.adoc

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "MicroProfile Language Support for Open Liberty Applications"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/kathrynkodama
+author_github: https://github.com/kathrynkodama
+seo-title: MicroProfile Language Support for Open Liberty Applications - OpenLiberty.io
+seo-description: Leverage the new MicroProfile Language Server along with existing VS Code tools to develop your Eclipse MicroProfile applications with Open Liberty.
+blog_description: "Leverage the new MicroProfile Language Server along with existing VS Code tools to develop your Eclipse MicroProfile applications with Open Liberty."
+---
+= MicroProfile Language Support for Open Liberty Applications
+Kathryn Kodama https://github.com/kathrynkodama
+
+The new MicroProfile Language Server provides added support for developing your Eclipse MicroProfile applications with Open Liberty. Earlier this month, the first release of the Language Server for Eclipse MicroProfile (https://github.com/eclipse/lsp4mp[Eclipse LSP4MP]), which is an open incubator project under the Eclipse Foundation, was published. The Tools for MicroProfile VS Code extension consumes this language server, providing feature-rich support for MicroProfile APIs and properties in VS Code.
+
+In this blog, we will showcase how the new MicroProfile Language Server, along with existing tooling for VS Code, makes it even easier to develop your Open Liberty applications.
+
+== Installing the extensions
+
+The following extensions are available on the VS Code Marketplace to support developing your Open Liberty applications with MicroProfile.
+
+- https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile[Tools for MicroProfile]: Language support for MicroProfile APIs and properties via the https://github.com/eclipse/lsp4mp[Language Server for Eclipse].
+
+- https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Open Liberty Tools]: Support for development with Open Liberty.
+
+- https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-starter-vscode-ext[MicroProfile Starter]: Quickly generate a MicroProfile Maven project.
+
```

Comment:
Offer an option for users to install all in one go with MicroProfile Extension Pack?

Reply:
Updated to: 
```
Install the https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.vscode-microprofile-pack[MicroProfile extension pack] from the VS Code Marketplace to support developing your Open Liberty applications with MicroProfile. We will use the following extensions from the MicroProfile extension pack:
```